apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -128,14 +132,22 @@ export default function CurriculumQuickAssign({updateSection, sectionCourse}) {
           </label>
         </div>
       </div>
-      {marketingAudience && isGradeBand && courseOfferings && (
+      {marketingAudience && !isPlOrHoc() && courseOfferings && (
```

Comment:
I realized that using a const instead of a function made the data inaccurate (lagged by 1 render) so I updated it.

Reply:
huh interesting! Do you know why? (It's okay if not -- this looks good too!)

Reply:
I couldn't quite wrap my head around it, but it felt like it had to do with the timing of rerenders in relation to the const being updated. But once it worked immediately as a function I stopped investigating😂